### DISCUSSION TITLE
Exclude files from _metadata.yml

### DISCUSSION THREAD

**Discussion by ma-riviere** [quarto-dev/quarto-cli#2594] [Q&A]
Hi,

Let's say I have a `blog` folder, within which are multiple subfolders, each containing a blog post.

Within that `blog` folder, I have an `index.qmd` file for the listings, and `_metadata.yml` file with YAML properties I want to apply to each of the blog posts.

How can I exclude the `index.qmd` of the `blog` folder from those properties ? I want the properties to apply to each blog post but not the index page that lists them. For context, the property I am trying to not apply to the `index.qmd` is an html file that I add using `include-before-body`.

Thank you :)

**Comment by dragonstyle** [comment]
I think your best bet would be to put all of your posts in a subfolder and put the `_metadata.yml` file for them in that subfolder.

**Reply by ma-riviere** [reply]
Yeah, that was my first thought too. But I wanted to see if there was a cleaner way to do this (i.e. without messing with my existing file structure).
